src/megachatapi.h

```diff
@@ -3842,7 +4148,8 @@ class MegaChatRoom
         CHANGE_TYPE_CLOSED              = 0x20, /// The chatroom has been left by own user
         CHANGE_TYPE_OWN_PRIV            = 0x40, /// Our privilege level has changed
         CHANGE_TYPE_USER_STOP_TYPING    = 0x80, /// User has stopped to typing. \see MegaChatRoom::getUserTyping()
-        CHANGE_TYPE_ARCHIVE             = 0x100 /// Archived or unarchived
+        CHANGE_TYPE_ARCHIVE             = 0X100, /// Archived or unarchived
+        CHANGE_TYPE_CHAT_MODE           = 0x400, /// User has set chat mode to private
```

Comment:
Why not `0x200`? Is there another change-type already in use in `develop`, or it's simply a typo due to copy&paste the change-type from the `MegaChatListItem`??

Reply:
CHANGE_TYPE_CALL is 0x200